CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased](https://github.com/idealista/kafka-role/tree/develop)
 
+- *[#23](https://github.com/idealista/kafka-role/issues/23) Using Vagrant hostmanager instead of Landrush* @dortegau
 - Picking up Xms correctly from KAFKA_HEAP_OPTS environment variable @didumgai

```

Comment:
Maybe we could add the link to the PR as well

`- *[#22](https://github.com/idealista/kafka-role/pull/22) Picking up Xms correctly from KAFKA_HEAP_OPTS environment variable* @didumgai`

Reply:
done :wink: